docs/development/core/server/kibana-plugin-server.irouter.options.md

```diff
@@ -0,0 +1,13 @@
+<!-- Do not edit this file. It is automatically generated by API Documenter. -->
+
+[Home](./index.md) &gt; [kibana-plugin-server](./kibana-plugin-server.md) &gt; [IRouter](./kibana-plugin-server.irouter.md) &gt; [options](./kibana-plugin-server.irouter.options.md)
+
+## IRouter.options property
+
+Register a route handler for `OPTIONS` request.
+
+<b>Signature:</b>
+
+```typescript
+options: <P extends ObjectType, Q extends ObjectType, B extends ObjectType>(route: RouteConfig<P, Q, B, 'options'>, handler: RequestHandler<P, Q, B>) => void;

```

Comment:
Is there are real use case when a plugin wants to declare a route handler for `options` request?

Reply:
Not that I know of, but I implemented it only for completion because of this discussion: https://github.com/elastic/kibana/pull/50783#discussion_r346995159

But I just saw your comment in there to avoid extending the API. Reverting those changes then :)